config/core/configmaps/features.yaml

```
@@ -60,3 +60,6 @@ data:
   # For more details: https://github.com/knative/eventing/issues/7739
   cross-namespace-event-links: "disabled"
 
+  # ALPHA feature: The new-apiserversource-filters flag allows you to use the new `filters` field
+  # in APIServerSource objects with its rich filtering capabilities.
+  new-apiserversource-filters: "enabled"
```

nit: normally, for alpha features we start with the feature flag disabled by default
```suggestion
  new-apiserversource-filters: "disabled"
```

---

Changed to disabled here and on the defaults struct.